hack/make-release-artifacts.sh

```diff
@@ -25,46 +25,47 @@ if [[ ! -d "${bin_dir}" ]]; then
   exit 1
 fi
 
-krew_tar_archive="krew.tar.gz"
-krew_exe="krew.exe"
-
-# copy license
-cp -- "${SCRIPTDIR}/../LICENSE" "./${bin_dir}"
-
-# create a out/krew.exe convenience copy
-if [[ -x "./${bin_dir}/krew-windows_amd64.exe" ]]; then
-  cp -- "./${bin_dir}/krew-windows_amd64.exe" "./out/krew.exe"
+checksum_cmd="shasum -a 256"
+if hash sha256sum 2>/dev/null; then
+  checksum_cmd="sha256sum"
 fi
+checksum_sed=""
 
-# consistent timestamps for files in bindir to ensure consistent checksums
 while IFS= read -r -d $'\0' f; do
-  echo "modifying atime/mtime for $f"
-  TZ=UTC touch -at "0001010000" "$f"
-  TZ=UTC touch -mt "0001010000" "$f"
-done < <(find "${bin_dir}" -print0)
+  archive_dir="$(mktemp -d)"
+  cp "$f" "${archive_dir}"
+  cp -- "${SCRIPTDIR}/../LICENSE" "${archive_dir}"
+  archive="$(basename "$f" .exe).tar.gz"
+  echo >&2 "Creating ${archive} archive."
+  (
+    cd "${archive_dir}"
+    # consistent timestamps for files in archive dir to ensure consistent checksums
+    TZ=UTC touch -amt "0001010000" ./*
+    tar --use-compress-program "gzip --no-name" -cvf "${SCRIPTDIR}/../out/${archive}" ./*
+  )
 
-echo >&2 "Creating ${krew_tar_archive} archive."
-(
-  cd "${bin_dir}"
-  tar --use-compress-program "gzip --no-name" -cvf "${SCRIPTDIR}/../out/${krew_tar_archive}" ./*
-)
+  # create sumfile
+  sumfile="out/${archive}.sha256"
+  checksum="$(eval "${checksum_cmd[@]}" "out/${archive}" | awk '{print $1;}')"
+  echo >&2 "${archive} checksum: ${checksum}"
+  echo "${checksum}" >"${sumfile}"
+  echo >&2 "Written ${sumfile}."
 
-checksum_cmd="shasum -a 256"
-if hash sha256sum 2>/dev/null; then
-  checksum_cmd="sha256sum"
-fi
+  # prepare krew manifest sed
+  checksum_sed="${checksum_sed};s/KREW_$(basename "$f" .exe | sed 's/krew-\(.*\)/\U\1/')_CHECKSUM/${checksum}/"
```

Comment:
Let's make the intent clearer by using `tr` here:
```suggestion
  checksum_sed="${checksum_sed};s/$(tr "[[:lower:]-]" "[[:upper:]_]" <<< ${name}
)_CHECKSUM/${checksum}/"
```

Reply:
Ah ah, I did hesitate between both.